corehq/apps/userreports/pillow.py

```diff
@@ -57,9 +57,18 @@ def rebuild_table(self, table):
         table.rebuild_table()
 
     def python_filter(self, doc):
-        # filtering is done manually per indicator set change_transport
+        # filtering is done manually per indicator see change_transport
         return True
 
+    def change_trigger(self, changes_dict):
```

Comment:
(what's strange is that the original ticket claims the second case wasn't working - though i did say that i didn't necessarily believe it)